terminus-ui/scss/helpers/test-sass.md

```diff
@@ -22,24 +22,26 @@ Think SASS… @include everything (describe, test, etc.)
 
 Each helper file has a .spec.scss file that imports both `true` and the corresponding helper .scss file
 
+
 ## Yarn
 
 If you simply run `yarn run test`, the test doesn’t run, saying there are no changes. Essentially the _watch functionality doesn’t work_.
 
-Workaround (recommended): 
+Workaround (recommended):
 1. Select ‘p’ to run by regex of file name
 1. Type 'sass' and hit enter
 1. Every time you need to run the test again, go to the terminal and hit enter
 
-If you run `yarn run test` and select ‘a’ to run all tests, or if you run `yarn run test:ci:local` it will test the scss and catch errors and display them in the terminal. However, it does not show up in the istanbul report.
+If you run `yarn run test` and select ‘a’ to run all tests, or if you run `yarn run test:ci:local` it will test the scss and catch errors
+and display them in the terminal. However, it does not show up in the istanbul report.
```

Comment:
Looks like extra line here, or is it just pr display issue?

Reply:
35/36 are a paragraph split over two lines to avoid going over the max line length. There should be 2 empty lines below that before the next header. (markdown lint rules coming soon 😬)